frontend/src/components/TableView/TableView.tsx

```diff
@@ -92,7 +95,7 @@ export const TableView = <T extends MRT_RowData>({
       density: 'compact',
     },
     initialState: {
-      columnVisibility: { id: false },
+      columnVisibility: hiddenColumns, // HERE ERalfnkJAKFÖLCN!!!!!!!!!!!!!!!
```

Comment:
Also what :D

Reply:
:D I made mark that I found where the magic happens. Didn't want to lose it